view/base.js

```diff
@@ -338,8 +341,8 @@ if ( typeof kintInitialized === 'undefined' ) {
 			, shiftKey = e.shiftKey
 			, i = kint.currentPlus;
 
-
-		if ( keyCode === 68 ) { // 'd' : toggles navigation on/off
+		if ( keyCode === 68 || (keyCode === 74 && j_toggles) ) { // 'd' or 'j' : toggles navigation on/off
```

Comment:
The boolean just says if keyboard navigation is active already or not, so I'm about to push it with the name _keyboardNavigationIsActive_